test/integration/resize_test.go

```diff
@@ -33,16 +33,9 @@ var _ = Describe("vary VM parameters: memory cpus, disk", func() {
 		})
 
 		It("check VM's disk size", func() {
-			switch runtime.GOOS {
-			case "linux":
-				out, err := RunOnHostWithPrivilege("virsh", "vol-info", "crc.qcow2", "--pool", "crc")
-				Expect(err).NotTo(HaveOccurred())
-				Expect(out).Should(MatchRegexp(`Capacity:[\s]*31.00 GiB`))
-			case "windows":
-				out, err := RunOnHost("powershell", "Get-VHD", "-Path", "C:/Users/crcqe/.crc/machines/crc/crc.vhdx")
-				Expect(err).NotTo(HaveOccurred())
-				Expect(out).Should(MatchRegexp(`Size[\s]*:[\s]*3328\d{7}`)) // 31GiB = 33285996544B
-			}
+			out, err := SendCommandToVM("df -h | grep sysroot")
```

Comment:
`df -h /sysroot` will do almost exactly the same. `df -h /sysroot | tail -1` should be equivalent to this `grep sysroot`

Reply:
Can use that, thanks!